src/libs/utils.ts

```diff
@@ -104,14 +119,24 @@ function stripEnd(s: string, c: string) {
   return s.slice(0, i + 1);
 }
 
-export async function getObligations(connection: Connection, config: Config, lendingMarket) {
-  const resp = await connection.getProgramAccounts(new PublicKey(config.programID), {
+export function getProgramIdForCurrentDeployment() {
+  if (process.env.APP == "beta") {
+    return 'BLendhFh4HGnycEDDFhbeFEUYLP4fXB5tTHMoTX8Dch5';
+  } else if (process.env.APP == "production") {
+    return 'So1endDq2YkqhipRh3WViPa8hdiSpxWy6z3Z6tMCpAo';
+  }
```

Comment:
There's also devnet -  ALend7Ketfx5bxh6ghsCDXAoDrhvEmsXT3cynB6aPLgx

nit: Easier to read as an object
```
{
  'staging': 'ALend7Ketfx5bxh6ghsCDXAoDrhvEmsXT3cynB6aPLgx',
  'beta': 'BLendhFh4HGnycEDDFhbeFEUYLP4fXB5tTHMoTX8Dch5,
  'production': 'So1endDq2YkqhipRh3WViPa8hdiSpxWy6z3Z6tMCpAo',
}
```